charts/opentelemetry-collector/templates/daemonset.yaml

```diff
@@ -39,4 +39,7 @@ spec:
       {{- with .Values.dnsPolicy }}
       dnsPolicy: {{ . }}
       {{- end }}
+      {{- with .Values.dnsConfig }}
+      dnsConfig: {{ . }}
```

Comment:
@datsabk this will not render correctly.  You'll want to move to to the next line and then pipe it to `toYaml` and `nindent`.